pareto/tests/test_strategic_model.py

```diff
@@ -246,6 +250,10 @@ def test_strategic_model_build_units_scaled_units_consistency(build_strategic_mo
                 else:
                     assert_units_consistent(condata)
 
+    # Iterate through all Expressions
+    for e in m.component_objects(Expression):
+        assert_units_consistent(e)
+
```

Comment:
Should we also check unit consistency for constraints as well?

Reply:
@JavalVyas2000 checking for unit consistency is already done in this test - look above the code that you highlighted in this comment.